src/datapath.jl

```diff
@@ -79,6 +79,17 @@ function TSdataset(filepath::String)
     end
 end
 
+function load_ts_file(fpath)
+    data = open(fpath) do input
+          readuntil(input, "@data")
```

Comment:
It's good practice and coding habit to add 
* docstrings describing what the function does,
* short in-line comments describing what the different parts of the function do, e.g. the regex expression in line 88) and `readuntil()` function call in line 84.

Reply:
I will push doctoring soon:)